server/src/transfer-cli.ts

```diff
@@ -0,0 +1,196 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as commandLineArgs from 'command-line-args';
+import * as ora from 'ora';
+import * as request from 'request-promise-native';
+
+// tslint:disable-next-line:no-require-imports no-any
+const commandLineUsage = require('command-line-usage') as any;
+
+const argDefs = [
+  {
+    name: 'help',
+    alias: 'h',
+    type: Boolean,
+    description: 'Print this help text',
+  },
+  {
+    name: 'token',
+    alias: 't',
+    type: String,
+    description:
+        'GitHub access token to perform this transfer. Must have the "repo" scope.',
+  },
+  {
+    name: 'force',
+    alias: 'f',
+    type: Boolean,
+    defaultValue: false,
+    description: 'Force transfer to happen',
+  },
+  {
+    name: 'from',
+    type: String,
+    description: 'Org to move these repos from',
+  },
+  {
+    name: 'to',
+    type: String,
+    description: 'Org to move these repos to',
+  },
+];
+
+/**
+ * Input is a list of repos (1 per line)
+ */
+function readInput(): Promise<string[]> {
+  return new Promise((resolve) => {
+    let input = '';
+    process.stdin.setEncoding('utf8');
+
+    process.stdin.on('readable', () => {
+      input += process.stdin.read() || '';
+    });
+
+    process.stdin.on('end', () => {
+      resolve(input.split('\n').filter((x) => x !== ''));
+    });
+  });
+}
+
+/**
+ * Returns whether or not the repo uses GitHub pages.
+ */
+function hasGitHubPages(
+    org: string, repo: string, token: string): Promise<boolean> {
+  return new Promise((resolve, reject) => {
+    request
+        .get({
+          url: `https://api.github.com/repos/${org}/${repo}`,
+          headers: {
+            'Accept': 'application/json',
+            'Authorization': `token ${token}`,
+            'User-Agent': 'Project Health Bot',
+          },
+          json: true,
+        })
+        .then((result) => {
+          // Check that this wasn't redirected.
+          if (result.owner.login !== org) {
+            reject(`Error: ${repo} was not found in ${org} and instead in ${
+                result.owner.login}`);
+            return;
+          }
+          if (result.has_pages) {
+            console.log(`[Warning] ${
+                repo} uses GitHub pages. This will not be transferred.`);
+          }
+          resolve(result.has_pages);
+        })
+        .catch((_err) => {
+          reject(`Error: ${org}/${repo} not found`);
```

Comment:
Check if 404 ? What it it's a token error. Or just show the msg

Reply:
Done.